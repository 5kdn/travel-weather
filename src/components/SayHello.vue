<script setup lang="ts">
const props = defineProps({
  name: String,
})
</script>

<script lang="ts">
export default {}
</script>

<template>
  <h1>Hello, {{ props.name }}!</h1>
</template>
