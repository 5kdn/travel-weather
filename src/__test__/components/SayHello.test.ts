import { mount } from '@vue/test-utils'
import { test, expect } from 'vitest'
import SayHello from '@/components/SayHello.vue'

test('SayHelloYoshio', () => {
  expect(SayHello).toBeTruthy()
  const wrapper = mount(SayHello, {
    props: {
      name: 'Yoshio',
    },
  })

  expect(wrapper.text()).toContain('Hello, Yoshio!')
  expect(wrapper.html()).toMatchSnapshot()
})
