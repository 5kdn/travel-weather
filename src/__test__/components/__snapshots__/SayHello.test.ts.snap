// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`SayHelloYoshio 1`] = `"<div>Hello, Yoshio!</div>"`;
